### DISCUSSION TITLE
Cant to play bot

### DISCUSSION THREAD

**Discussion by Irnasaid19** [eritislami/evobot#931] [Q&A]
(node:24711) UnhandledPromiseRejectionWarning: Error: Error: Cannot find module '/rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/@discordjs/opus/prebuild/node-v83-napi-v3-linux-x64-glibc-2.23/opus.node'
Require stack:
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/@discordjs/opus/lib/index.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/node_modules/prism-media/src/util/loader.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/node_modules/prism-media/src/opus/Opus.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/node_modules/prism-media/src/opus/index.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/node_modules/prism-media/src/index.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/client/voice/player/BasePlayer.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/client/voice/player/BroadcastAudioPlayer.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/client/voice/VoiceBroadcast.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/client/voice/ClientVoiceManager.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/client/Client.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/index.js
- /app/index.js
Error: Cannot find module 'node-opus'
Require stack:
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/node_modules/prism-media/src/util/loader.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/node_modules/prism-media/src/opus/Opus.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/node_modules/prism-media/src/opus/index.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/node_modules/prism-media/src/index.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/client/voice/player/BasePlayer.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/client/voice/player/BroadcastAudioPlayer.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/client/voice/VoiceBroadcast.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/client/voice/ClientVoiceManager.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/client/Client.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/index.js
- /app/index.js
Error: Cannot find module 'opusscript'
Require stack:
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/node_modules/prism-media/src/util/loader.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/node_modules/prism-media/src/opus/Opus.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/node_modules/prism-media/src/opus/index.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/node_modules/prism-media/src/index.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/client/voice/player/BasePlayer.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/client/voice/player/BroadcastAudioPlayer.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/client/voice/VoiceBroadcast.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/client/voice/ClientVoiceManager.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/client/Client.js
- /rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/index.js
- /app/index.js
    at Object.loader [as require] (/rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/node_modules/prism-media/src/util/loader.js:12:9)
    at loadOpus (/rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/node_modules/prism-media/src/opus/Opus.js:17:17)
    at new OpusStream (/rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/node_modules/prism-media/src/opus/Opus.js:46:10)
    at new Decoder (/rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/node_modules/prism-media/src/opus/Opus.js:183:1)
    at AudioPlayer.playOpusStream (/rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/client/voice/player/BasePlayer.js:73:23)
    at VoiceConnection.play (/rbd/pnpm-volume/ba449c5a-d495-40b6-8a29-8489323b2d3f/node_modules/discord.js/src/client/voice/util/PlayInterface.js:75:28)
at play (/app/include/play.js:61:8)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:24711) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:24711) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

(node:24711) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:24711) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by Velosofy** [comment]
I'm no good, but did you try to nuke the repo, reclone and reinstall ?

**Reply by Irnasaid19** [reply]
Ok i try to reclone this repo. Thanks, sir
